controllers/storagecluster/cephtoolbox.go

```diff
@@ -48,6 +48,13 @@ func (r *StorageClusterReconciler) ensureToolsDeployment(sc *ocsv1.StorageCluste
 		return err
 	}
 
+	specifiedResourceRequirements, specified := sc.Spec.Resources["rook-ceph-tools"]
+	// if specified resource requirements is present but empty, the intention is to have no resource requirements
+	if specified && isResourceNonEmpty(specifiedResourceRequirements) {
```

Comment:
Can't we call getDaemonResources("toolbox", sc)
This merge logic is already present inside getDaemonResources